libnmstate/netapplier.py

```diff
@@ -156,7 +157,18 @@ def _apply_ifaces_state(
                     con_profiles=ifaces_add_configs + ifaces_edit_configs,
                 )
             if verify_change:
-                _verify_change(desired_state)
+                attempts_left = 5
+                retry = True
+                while(retry):
```

Comment:
You might also be able to write it like to avoid a possible endless loop:

```python
for _ in range(5):
    try:
        _verify_change(desired_state)
        break
    except NmstateVerificationError:
        time.sleep(1)
else:
    _verify_change(desired_state)
```

Reply:
This is better. Thanks!